website/docs/r/redis_cache.html.markdown

```diff
@@ -91,7 +91,7 @@ A `redis_configuration` block supports the following:
 
 * `maxfragmentationmemory_reserved` - (Optional) Value in megabytes reserved to accommodate for memory fragmentation. Defaults are shown below.
 
-* `rdb_backup_enabled` - (Optional) Is Backup Enabled? Only supported on Premium SKU's.
+* `rdb_backup_enabled` - (Optional) Is Backup Enabled? Only supported on Premium SKU's. If enabled, `rdb_storage_connection_string` must also be set.
```

Comment:
We should make this a note so it stands out

Reply:
updated to be a -> **NOTE:**